File: CSharp/IndexMultiplier.cs
using System.Linq;

namespace CSharp
{
    // Return the sum of all items in an array, where each item is multiplied by its index (zero-based). 
    // For empty arrays, return 0.
    // https://edabit.com/challenge/3Efavz8YmSBia4p8s
    public static class IndexMultiplier
    {
        public static int MultiplyIndex(int[] arr) => arr.Select((num, i) => num * i).Aggregate(0, (a, b) => a + b);
    }
}

File: python/index_multiplier.py
"""
Index Multiplier

Return the sum of all items in a list, where each item is multiplied by its index (zero-based). For empty lists, return 0.

https://edabit.com/challenge/82BssjYWrugBjRsux
"""
from typing import List


def index_multiplier(lst: List[int]) -> int:
    return sum(num * i for i, num in enumerate(lst))
